### ISSUE TITLE
For beginners, submodules are tricky to navigate, recommend a bit more readme docs on this

### ISSUE THREAD

**Issue by truedat101** [MycroftAI/mycroft-skills#389]
Submodules are tough for git beginners, and even tough for people who use git a lot but don't necessarily use submodules often.  At the very lease, it might be helpful to explain the top level of the doc:

git submodule init
git submodule update

or whatever other preferred commands are relevant to get going.  I'm teaching my Son to code on this mycroft, and he struggled when he it this problem.

**Comment by forslund** [comment]
You are absolutely correct, submodules are quite tricky. To make this a bit easier there's now a tool `msk` which can add and update skills instead of manually running submodule commands. However the use of the tool should be documented in the Readme.
